v0.5/tools/submission/submission-checker.py

```diff
@@ -0,0 +1,238 @@
+"""
+A checker for mlperf inference submissions
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import collections
+import json
+import logging
+import os
+import re
+import sys
+import time
+
+# pylint: disable=missing-docstring
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("main")
+
+VALID_MODELS = ["ssd-small", "ssd-large", "mobilenet", "resnet", "gnmt"]
+REQUIRED_DIRECTORIES = ["code", "results", "measurements", "systems"]
+REQUIRED_PERF_FILES = ["mlperf_log_accuracy.json", "mlperf_log_summary.txt", "mlperf_log_detail.txt"]
+REQUIRED_ACC_FILES = REQUIRED_PERF_FILES + ["accuracy.txt"]
+REQUIRED_MEASURE_FILES = ["mlperf.conf", "README.md"]
```

Comment:
`mlperf.conf` should be `user.conf` according to https://github.com/mlperf/policies/blob/master/submission_rules.adoc

Reply:
I think it wants both so I added a empty user.conf